### PR TITLE
Replace thumbnail image

### PR DESCRIPTION
**Pull request by precious-onyenaucheya-ons**
### What is the context of this PR?

This PR fixes https://github.com/ONSdigital/ons-service-manual-website/issues/400
The thumbnail  image for document list and card component was replaced to have a darker fill color('--ons-color-image-placeholder').

### How to review this PR

Check that all thumbnail images in the project has been updated
Check that all visual tests is updated accordingly
 
### Checklist

This needs to be completed by the person raising the PR.

-   [x] I have selected the correct Assignee
-   [x] I have linked the correct Issue

### CODE REVIEW

File: backstop_data/bitmaps_reference/ds-vr-test__patterns_download-resources_example-download-resources_0_document_0_desktop.png
Comment (alessioventuriniAND) [inline backstop_data/bitmaps_reference/ds-vr-test__patterns_download-resources_example-download-resources_0_document_0_desktop.png:1]: The border of the portrait placeholder image needs to have the same color as the new one and the focus state background color needs be changed to the new one.

**Review by alessioventuriniAND — CHANGES_REQUESTED** [review]

File: backstop_data/bitmaps_reference/ds-vr-test__patterns_download-resources_example-download-resources_0_document_0_desktop.png
Comment (precious-onyenaucheya-ons) [reply backstop_data/bitmaps_reference/ds-vr-test__patterns_download-resources_example-download-resources_0_document_0_desktop.png:1]: updated

**Review by alessioventuriniAND — APPROVED** [review]

**Review by rmccar — CHANGES_REQUESTED** [review]
Few minor things

**Comment by rmccar** [comment]
Screenshot 2024-08-08 at 10 54 55

Focus state looks slightly off, adding "!important" after "outline-offset: 0" on line 135 in the document list css file will solve this

File: src/components/document-list/document-list.scss
[the remark this answers is not in the document]
Comment (precious-onyenaucheya-ons) [reply src/components/document-list/document-list.scss:1]: done

File: src/components/document-list/document-list.scss [hunk 1]
```diff
@@ -128,7 +128,7 @@
         width: 100%;
 
         &:focus {
-            background-color: var(--ons-color-image-placeholder) !important;
```
[the remark this answers is not in the document]
Comment (precious-onyenaucheya-ons) [reply src/components/document-list/document-list.scss:131]: removed

**Pull request merged**